src/kOS.Safe/Encapsulation/ScalarValue.cs

```diff
@@ -67,12 +72,6 @@ public static ScalarValue Create(object value)
                 return new ScalarIntValue((int)value);
             }
 
-            var scalarValue = value as ScalarValue;
-            if (scalarValue != null)
-            {
-                return Create(scalarValue.Value);
-            }
-
```

Comment:
I actually tested it both ways, and it was a benefit but not a huge benefit.  I figured even a small gain would be worth it.